spec/hydra/pcdm/models/object_spec.rb

```diff
@@ -14,6 +14,45 @@
     end
   end
 
+  context 'when aggregated by other objects' do
+
+    before(:all) do
+      @collection1 = Hydra::PCDM::Collection.create
```

Comment:
Could you add a comment to the code that states this? It'd be good to see in the `before(:all)` block that this style was chosen because it has a significant impact on test performance.